flask_mongorest/resources.py

```diff
@@ -199,15 +200,24 @@ def serialize_field(self, obj, **kwargs):
         else:
             return self.serialize(obj, **kwargs)
 
+    def _subresource_class(self, obj):
```

Comment:
Even though this is an internal method, we should add a docstring because it's not immediately clear what it's doing (explain `default_child_resource_document` etc.).